### DISCUSSION TITLE
Playwright engine: core web vitals have disappeared

### DISCUSSION THREAD

**Discussion by RachaelBooth** [artilleryio/artillery#2236] [Artillery]
I have a small set of tests using the playwright engine. When I run them, I no longer get core web vitals in the report (i.e. CLS, LCP, etc) - is anyone else finding this, or know how I can get them back?

Not sure if I've missed some config - they're still mentioned in the docs, and I've not changed any code since I last ran them and did see those in the report (which was not long ago - yesterday or early this morning I think), but I've seen that artillery-engine-playwright seems to have had an update recently.

**Comment by bernardobridge** [comment]
Hi @RachaelBooth 👋 - thanks for the report . There are updates to the `artillery-engine-playwright` in progress, but shouldn't be affecting you, as these are just on `main` of Artillery right now and aren't published anywhere yet.

Can you please provide a bit more information:
- What Artillery version are these tests running in? (you can use `artillery --version` to determine that)
- Are they running in a CI environment, or locally, and with or without Fargate?
- Do you still get other metrics? Could you perhaps share the output you're getting from Artillery?

Thank you!

**Reply by RachaelBooth** [reply]
Artillery version is 2.0.0-37
I've seen the same thing running locally and with Fargate - I initially thought it was Fargate problem because the first time they weren't in the output was also the first time I ran with Fargate, but since then have got the same either way.
I still get vusers metrics, and http_request and domcontentloaded metrics (with extended metrics turned on).

**Reply by RachaelBooth** [reply]
By the way, to clarify what I said about updates: it looks like the artillery package specifies "latest" as the version for it's artillery-engine-playwright dependency, and npm is showing an update to that yesterday (i.e. I'm not meaning that the github repository is showing work going on, but that there's something published that may be being used). I can't see any other reason for the change in behaviour, I am genuinely running the same code except that I used run-fargate once (but I'm seeing the behaviour with both). 

I have tried going back a version of artillery but see the same problem.

**Reply by bernardobridge** [reply]
Hi @RachaelBooth 👋 . The `artillery-engine-playwright` only got an update to its `canary` version, if you look at the `latest` tag (which is what `artillery` uses), it still hasn't changed (it will only be updated later today).

I haven't been able to reproduce the issue both on older versions, and on the new code I'm working on. If it's also not working on older Artillery versions, I'm not sure the issue is with Artillery? Perhaps something with your application is different? You can maybe try to run the test without headless locally and see if there's anything wrong with the test itself now.

The only other option I can think of is that something in your `ms-playwright` browsers path (which the Playwright engine uses) has changed for other reasons, and is affecting your Artillery runs. You could try deleting those browsers (if you're not using them for something else), and then reinstalling Artillery again (which should reinstall the browsers).

**Comment by up201306506** [comment]
Potentially related to your problem, but I can't tell without seeing your yaml: When i was running some playwright tests that potentially a couple different domains at random (Don't worry, they're in the same webserver), the Web Vitals would NOT be tracked except for the domain in the `target` value.

So I just did

```
config:
  target: https:/
```

as suggested by some obscure forgotten old thread, and problem solved. Well, sorta. I had to include `aggregateByName: true` on the playwright engine as well.
